src/cljs/athens/components.cljs

```diff
@@ -14,13 +14,14 @@
 
 (defn todo-on-click
   [uid from-str to-str]
-  (let [current-block-content (:block/string (db/get-block [:block/uid uid]))]
-    (dispatch [:transact [{:block/uid    uid
-                           :block/string (clojure.string/replace
-                                           current-block-content
-                                           from-str
-                                           to-str)
-                           :edit/time    (now-ts)}]])))
```

Comment:
`:edit/time` is our own property, it's not automatically managed by DB.
There is no DEFAULT of SQL in Datascript.

Reply:
Ah, I see `:edit/time` can be attached to pages and then it can be used to sort the pages by time, don't know how TODO uses this will check this and update.